src/serializatorAndDeserializator/Deserializator.java

```diff
@@ -6,9 +6,14 @@
 import java.io.ObjectInputStream;
 
 public class Deserializator {
+    private String path;
 
-        public java.lang.Object deserialize(String path) {
-            java.lang.Object o = null;
+    public Deserializator(String path){
+        this.path = path;
+    }
+
+        public Object deserialize(String path) {
+            Object o = null;
             FileInputStream fis = null;
             ObjectInputStream ois = null;

```

Comment:
Remove redundant space before "deserialize" method. It should be on the same level like constructor